Samples/iOS-Swift/iOS-Swift6/ViewController.swift

```diff
@@ -0,0 +1,10 @@
+import Sentry
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
Can we change the sample to SwiftUI? I guess most people using Swift 6 will also use SwiftUI.

Reply:
SwiftUI would have been less code 😄